client/src/services/localStorage.js

```diff
@@ -104,7 +104,7 @@ const getChromeStorageLocal = async (key) => {
 	}
 };
 
-const clearChromeStorageLocal = () => chrome.storage.local.clear();
+const clearChromeStorageLocal = async () => await chrome.storage.local.clear();
```

Comment:
Good one, I didn't take into account this would cause any strang behaviour. Given it would return result as `Promise`